cypress/integration/rendering/gantt.spec.js

```diff
@@ -490,4 +490,40 @@ describe('Gantt diagram', () => {
       {}
     );
   });
+
+  it('should render when there is a dateRange', () => {
+    imgSnapshotTest(
+      `
+      ---
+      displayMode: compact
+      ---
+      gantt
+        title GANTT compact
+        dateFormat  HH:mm:ss
+        dateRange   12:30:00,13:30:00
+        axisFormat  %Hh%M
```

Comment:
Can you add the `imgSnapshotTest`s, for the other scenarios of dateRange?

- Actual date
- Only start
- Only end

Reply:
Certainly, but how do these work?  Where are the images produced so that I can check them?  I suspect that without altering the tasks, the display is going to be messed up, with the task bars invading the margins, but I don't know how to produce any diagram output from this repo.

Reply:
To verify snapshots:

```sh
pnpm e2e
open cypress/snapshots
```

Can you also add this into the contribution docs if it's not present there?